.sync.yml

```diff
@@ -0,0 +1,53 @@
+---
+.travis.yml:
+  docker_sets:
+    - set: docker/centos-7
+    - set: docker/ubuntu-14.04
+  docker_defaults:
+    bundler_args: ""
+    script: bundle exec rake task_acceptance
+  bundler_args: ""
```

Comment:
Need this to avoid default `--without-system-tests` because all of our matrix cells depend on that.